lib/js_rails_routes/language/javascript.rb

```diff
@@ -11,7 +11,13 @@ class JavaScript < Base
           var query = [];
           for (var param in params) if (Object.prototype.hasOwnProperty.call(params, param)) {
             if (keys.indexOf(param) === -1) {
-              query.push(param + "=" + encodeURIComponent(params[param]));
+              if (Array.isArray(params[param])) {
+                for (var value of params[param] as Value[]) {
```

Comment:
Please remove type assertion, since it's JavaScript. :pray:

```suggestion
                for (var value of params[param]) {
```

Reply:
Thank you for your feedback. I have made the suggested changes.